setup.cfg

```diff
@@ -1,59 +1,65 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[build_docs]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
 [tools:pytest]
-minversion = 2.2
-norecursedirs = build docs/_build
-remote_data_strict = true
-
-[ah_bootstrap]
-auto_use = True
+testpaths = "pypi" "docs"
+astropy_header = true
+doctest_plus = enabled
+text_file_format = rst
+addopts = --doctest-rst
 
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-exclude = __init__.py, setup_package.py, conf.py, setup.py, ah_bootstrap.py, astropy_helpers, version.py, ci-helpers conftest.py
+exclude = __init__.py, setup_package.py, conf.py, setup.py, version.py, ci-helpers conftest.py
 
 [pycodestyle]
 max-line-length = 110
 max-doc-length = 79
 exclude = extern,sphinx,*parsetab.py
 
 [metadata]
-package_name = pyvo
+name = pyvo
 description = Astropy affiliated package for accessing Virtual Observatory data and services
-long_description =
-author = the IVOA community
-author_email = sbecker@ari.uni-heidelberg.de
+long_description = file: README.rst
+author = the PyVO Developers
 license = BSD
+license_file = LICENSE.rst
 url = https://github.com/astropy/pyvo
 edit_on_github = False
 github_project = astropy/pyvo
 install_requires = astropy>=4.0 requests
 # version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
 version = 1.3.dev
 
+[options]
+setup_requires = setuptools_scm
+install_requires =
+    astropy>=4.0
+    requests
+python_requires = >=3.6
```

Comment:
if you support python 3.6, then you should test it. The oldest one in tox and the CI setup is 3.7. I would rather suggest to drop 3.6

Reply:
Yeah, Python 3.6 has reached EOL. 👍 to dropping it.

Reply:
Yes, and given that most upstream have dropped 3.7, too, you may want to consider dropping that one, too.

Reply:
Dropped 3.6 and 3.7